addon/models/osf-model.js

```diff
@@ -0,0 +1,50 @@
+/*
+  Common properties and behaviors shared by all OSF APIv2 models
+ */
+
+import Ember from 'ember';
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+    links: DS.attr('links'),
+    embeds: DS.attr('embed'),
+
+    relationshipLinks: Ember.computed.alias('links.relationships'),
```

Comment:
Also I want to have a placeholder in place in case we discover a way to not shove relationships in the links field.